src/scores/categorical/contingency_impl.py

```diff
@@ -1014,27 +1077,38 @@ def symmetric_extremal_dependence_index(self) -> xr.DataArray:
 
 class BinaryContingencyManager(BasicContingencyManager):
     """
+    See https://scores.readthedocs.io/en/stable/tutorials/Binary_Contingency_Scores.html for
+    a detailed walkthrough showing the use in practice.
```

Comment:
Given we have been moving to relative links where possible, can/should this be a relative path?

Reply:
As above, not sure this is possible.